cms/djangoapps/contentstore/rest_api/v1/serializers/course_details.py

```diff
@@ -10,10 +10,10 @@
 class InstructorInfoSerializer(serializers.Serializer):
     """ Serializer for instructor info """
     name = serializers.CharField(allow_blank=True)
```

Comment:
Should this also be `required=False`?

Reply:
I'm assuming there has to at least be a name for instructor_info, though I'm not completely sure. The specific issue that courses have been running into is a missing organization field.